aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -32,6 +34,15 @@ module aptos_framework::account {
         new_authentication_key: vector<u8>,
     }
 
+    #[event]
+    struct KeyRotationToMultiPublicKey has drop, store {
```

Comment:
Can you add some short comments explaining the fields since they're not obviously easy to understand